_pkgdown.yml

```diff
@@ -22,8 +22,6 @@ footer:
   structure:
     left: [package, developed_by]
     right: built_with
-  components:
```

Comment:
I would leave this in and have it look like this:
```
components:
  logo: "<img src='favicon-16x16.png'>"
```
but do *not* add the `logo` element to the `left: [ package, developed_by]` section. 
this way it doesn't do anything here, but in the `pkgdown` branch, you can add `logo` to the `left:` component, and it'll take over from there.

Reply:
This is a good idea!